gradoop-temporal/src/test/java/org/gradoop/temporal/model/impl/operators/aggregation/functions/MinMaxTimeTest.java

```diff
@@ -117,15 +68,40 @@ public void testAggregationFunctions() throws Exception {
     assertEquals(PropertyValue.create(expectedMin), head.getPropertyValue(keyMin));
   }
 
+  /**
+   * Get parameters for this test. Those are
+   * <ol>
```

Comment:
You can set the start of a list like this, I think this would make it more correct, since java indices start with `0`.
Not necessary though, just personal preference
```suggestion
   * <ol start="0">
```